WrightTools/data/_kent.py

```diff
@@ -22,145 +22,61 @@
-        indicies.reverse()
-        for i in indicies:
-            dim_check = dims_unaccounted[i]
-            if d_equal[dim_check, dim_current]:
-                tolerance.append(dims[dim_check][1])
-                key.append(dims[dim_check][3])
-                dims_unaccounted.pop(i)
-        tolerance = max(tolerance)
-        dims_condensed.append([index, tolerance, units, key])
-    dims = dims_condensed
-    # which dimensions are scanned ----------------------------------------------------------------
-    # find
-    scanned = []
-    constant_list = []
-    for dim in dims:
-        name = dim[3]
-        index = dim[0]
-        vals = arr[index]
-        tolerance = dim[1]
-        if vals.max() - vals.min() > tolerance:
-            scanned.append([name, index, tolerance, None])
-        else:
-            constant_list.append([name, index, tolerance, arr[index, 0]])
-    # order scanned dimensions (..., zi, yi, xi)
-    first_change_indicies = []
-    for axis in scanned:
-        first_point = arr[axis[1], 0]
-        for i in range(len(arr[0])):
-            upper_bound = arr[axis[1], i] + axis[2]
-            lower_bound = arr[axis[1], i] - axis[2]
-            if upper_bound > first_point > lower_bound:
-                pass
-            else:
-                first_change_indicies.append(i)
-                break
-    scanned_ordered = [scanned[i] for i in np.argsort(first_change_indicies)]
-    scanned_ordered.reverse()
-    # return --------------------------------------------------------------------------------------
-    # package back into ordered dictionary of objects
-    scanned = collections.OrderedDict()
-    for axis in scanned_ordered:
-        key = axis[0][0]
-        obj = input_cols[key]
-        obj.label_seed = [input_cols[_key].label_seed[0] for _key in axis[0]]
-        scanned[key] = obj
-    constant = collections.OrderedDict()
-    for axis in constant_list:
-        key = axis[0][0]
-        obj = input_cols[key]
-        obj.label_seed = [input_cols[_key].label_seed[0] for _key in axis[0]]
-        obj[:] = axis[3]
-        constant[key] = obj
-    return list(scanned.values()), list(constant.values())
-
-
 # --- from function -------------------------------------------------------------------------------
 
 
-def from_KENT(filepaths, null=None, name=None, ignore=['wm'], delay_tolerance=0.1,
-              frequency_tolerance=0.5, verbose=True):
-    """Read data object from KENT files."""
+def from_KENT(filepaths, name=None, ignore=['wm'], delay_tolerance=0.1, frequency_tolerance=0.5,
+              parent=None, verbose=True):
+    """Create data object from KENT file(s).
+
+    Parameters
+    ----------
+    filepaths : string or list of strings
+        Filepath(s).
+    name : string (optional)
+        Unique dataset identifier. If None (default), autogenerated.
+    ignore : list of strings (optional)
+        Columns to ignore. Default is ['wm'].
```

Comment:
What is the reason for ignoring `wm` by default?

Reply:
it's something we've done from the beginning, because wm typically in complex ways that are not auto-recognizable in my naiive `discover_dimensions` routine (a lot of DOVE experiments)

Reply:
With WrightTools 3 variables, does it make sense to include it as a variable, even if it does not affect the dimensionality or get fed into discover_dimensensions by default?